packages/rest/src/rest.server.ts

```diff
@@ -190,6 +192,25 @@ export class RestServer extends Context implements Server, HttpServerLike {
     this._setupRequestHandler();
 
     this.bind(RestBindings.HANDLER).toDynamicValue(() => this.httpHandler);
+
+    // LB4's static assets serving router
+    const staticAssetsRouter = new StaticRoute(
+      (req: Request, res: Response) => {
+        return new Promise((resolve, reject) => {
+          const onFinished = () => resolve();
+          res.once('finish', onFinished);
+          this._routerForStaticAssets.handle(req, res, (err: Error) => {
+            if (err) {
+              return reject(err);
+            }
+            // Express router called next, which means no route was matched
+            return reject(new HttpErrors.NotFound());
```

Comment:
Let's be more verbose here. See how we deal with 404 errors now:

https://github.com/strongloop/loopback-next/blob/a2ab86d3a3c721f28159585ca4e87bd8504476c4/packages/rest/src/router/routing-table.ts#L183-L186